data/human/kestrel.txt

```diff
@@ -60,6 +67,7 @@ mission "Kestrel Testing"
 	
 	on complete
 		payment 2000000
+		"global: unlocked kestrel" ++
```

Comment:
Set instead of increment to make the suggested offer condition changes easier to handle. 
```suggestion
		set "global: unlocked kestrel"
```

Reply:
Honestly, I like being able to see how often I've unlocked the Kestrel. Much more useful than a mere true/false bit.

Reply:
Added an alternative `to offer` suggestion that would account for this condition being a counter.